expat/lib/xmltok.c

```diff
@@ -869,7 +870,7 @@ static const struct normal_encoding internal_little2_encoding
 
 #define BIG2_BYTE_TYPE(enc, p)                                                 \
   ((p)[0] == 0                                                                 \
-       ? ((struct normal_encoding *)(enc))->type[(unsigned char)(p)[1]]        \
+       ? SB_BYTE_TYPE(enc, p)                                                  \
```

Comment:
I think we need `p + 1` rather than `p` here

Reply:
Damn! Sorry about that, fix coming...